.gitignore

```diff
@@ -50,3 +50,4 @@ src/java/us/kbase/mobu/git.properties
 /a_SimpleModule_for_unit_testing_java/
 /a_SimpleModule_for_unit_testing_python/
 /a_SimpleModule_for_unit_testing_perl/
+/ASimpleModule_for_unit_testingPythonError/
```

Comment:
I'm not seeing where this gets created or referenced.

Reply:
https://github.com/kbase/kb_sdk_plus/blob/8383d43b829146b7efc09eade559d9008e88b6f5/src/java/us/kbase/mobu/tester/test/ModuleTesterTest.java#L119